src/ngraph/op/experimental/dyn_pad.hpp

```diff
@@ -0,0 +1,51 @@
+//*****************************************************************************
+// Copyright 2017-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include "ngraph/op/op.hpp"
+
+namespace ngraph
+{
+    namespace op
+    {
+        //// \brief Generic padding operation which takes padding below and above as dynamic shapes.
+        //// This is similar to existing Pad operation except padding values are dynamic and no interior padding.
```

Comment:
There should be three slashes on doc string comments, not four.

Interior padding no longer exists for `Pad` either, since #2590 was merged.